### DISCUSSION TITLE
Ein- und ausschalten von Layern auf mobil devices verbesserungswürdig

### DISCUSSION THREAD

**Discussion by ue-black** [e-netze/webgis-community#89] [General]
Hallo!

Wir haben das Feedback bekommen, dass Layer ein- und ausschalten auf mobil devices lästig sein kann, da man tatsächlich links daneben tippen muss. Wenn man auf den Radio oder Checkbox Button tippt, öffnet sich der Metadatenkatalog.

Siehe dieses Demonstrations-Video: 
https://github.com/gis-eni/webgis-forum/assets/127403099/462d8f90-b5b5-4847-98e7-4201b00242c1

Wir haben als Diensthandys im Land IPhone SE. Bei folgenden Typen wurde es getestet (rot = Fehler ; grün = funktioniert wenn man auf die Checkbox tippt)

![06-07-_2023_11-18-33](https://github.com/gis-eni/webgis-forum/assets/127403099/b44676ce-df89-4946-bd20-210b0e250a34)

Wie können wir die Usability verbessern?

**Comment by gis-eni** [comment]
Dem Metadaten Button weg geben? ;)

Grundsätzlich ist es so, dass man NICHT zwingend auf die Checkbox tippen muss und einen Layer oder eine Darstellungsvariante zu schalten. Die Checkbox dient eigentlich nur dazu, dass man weiß ob ein- oder ausgeschalten.
Gleich bei Drowdown-Layergruppen.

Man "tippt" eigentlich auf das Element, also auf den Text (überall hin, nur ich auf den Metadaten Button ;))
Das eigentlich eine gängige Praxis auf Mobilen Geräten.

**Reply by ue-black** [reply]
Als Desktop-GIS Anwender ist man halt konditioniert auf die Checkbox zu drücken um Layer ein- und auszuschalten. Beim Drücken auf den Text (oder Doppelklick) stecken dann andere Funktionen dahinter. Aber da das Feedback von einem GIS Power-User kommt, würde ich das so weiter geben und das wäre in dem Fall erledigt. Danke für die schnelle Rückmeldung!
